app/scripts/metamask-controller.js

```diff
@@ -2456,18 +2456,21 @@ export default class MetamaskController extends EventEmitter {
    * array if no accounts are permitted.
    *
    * @param {string} origin - The origin whose exposed accounts to retrieve.
+   * @param {boolean} suppressUnauthorized - suppresses the unauthorized error. defaults to `true`.
```

Comment:
Optional 2: 
What if we renamed `suppressUnauthorized` to `suppressUnauthorizedErrorCode`?

I'm proposing this because one of my first thoughts upon reading`suppressUnauthorized` is that the return value would not include unauthorized accounts (if this could be a thing). However, I have observed that `suppressUnauthorized` is specific to the error logic.

Reply:
I agree with Option 2 above ^ I am not a  big fan of passing bool as params but I guess in this case its the less painful path

Reply:
renamed to `suppressUnauthorizedError`